lib/common/ServiceInfo.ts

```diff
@@ -0,0 +1,46 @@
+import ServiceVersionModel from './models/ServiceVersionModel'; 
+
+/**
+ * Encapsulates the functionality to get the information about the service such as
+ * version info.
+ */
+export default class ServiceInfo {

```

Comment:
Design: I was imagining this class would be doing the fetching and caching, so that `Cas` and `Blockchain` don't have to know about the implementation details.

Reply:
Changed it to a ServiceInfoProvider.ts